shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate.mm

```diff
@@ -73,9 +74,6 @@ - (void)dealloc {
   for (dispatch_block_t unsubscribe in _notificationUnsubscribers) {
```

Comment:
You can delete this code, all of `dealloc`, `_notificationUnsubscribers` itself, and optionally the `addObserver:forSelector:` helper that will just become a single line. This unregistration hasn't been necessary since we dropped iOS 8 support.

Reply:
I was going to do all the unsubscribes separately, I didn't have a chance to put it out for review yet
https://github.com/jmagman/engine/commit/02a177c450f6ee7d1d187b418d971de1817cb7e3

Reply:
https://github.com/flutter/engine/pull/52772